Documentation/recovery_v3.md

```diff
@@ -0,0 +1,39 @@
+## Disaster recovery
+
+etcd is designed to withstand machine failures. An etcd cluster automatically recovers from temporary failures (e.g., machine reboots) and tolerates up to _(N-1)/2_ permanent failures for a cluster of N members. When a member permanently fails, whether due to hardware failure or disk corruption, it loses access to the cluster. If the cluster permanently loses more than _(N-1)/2_ members then it disastrously fails, irrevocably losing quorum. Once quorum is lost, the cluster cannot reach consensus and therefore cannot continue accepting updates.
+
+To recover from disastrous failure, etcd provides snapshot and restore facilities to recreate the cluster without data loss.
+
+### Snapshotting the keyspace
+
+Recovering a cluster first needs a snapshot of the keyspace from a functioning etcd node. Snapshots are taken with the `etcdctl snapshot save` command.  For example, the following command snapshots the keyspace to the file `snapshot.db`:
```

Comment:
will copy the member/snap/db just work if user has a dead cluster with one existing member?